packages/block-library/src/gallery/utils.js

```diff
@@ -36,17 +40,32 @@ export function getHrefAndDestination(
 			return {
 				href: image?.source_url || image?.url, // eslint-disable-line camelcase
 				linkDestination: IMAGE_LINK_DESTINATION_MEDIA,
+				lightbox: lightboxSetting?.enabled
+					? { enabled: false }
+					: undefined,
```

Comment:
```suggestion
				lightbox: undefined,
```

Why do we check `lightboxSettings.enabled`? Shouldn't lightbox always be undefined here?

Reply:
When lightbox is enabled by default, we need to set `lightbox: { enabled: false }` to disable lightbox and set other link options. Checkout [lightbox logic](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/image/image.js#L272-L286) in image block. If we use `lightbox: undefined`, Lightbox will remain active even for other link destinations.

Reply:
Thank you for the detailed explanation.

Until now, it worked without this additional processing because the conditional statement [here](https://github.com/WordPress/gutenberg/blob/d79d610a4d9e464d6bf583bdf6a140ff2ae262c3/packages/block-library/src/image/index.php#L63-L72) checked whether the link destination was none, even if the lightbox was enabled by default.

In any case, I think it makes sense to make it consistent with the logic of the Image block and explicitly disable it if `enabled` is active by default.